docs/development.md

```diff
@@ -188,6 +188,13 @@ All other test suites will be run once and only once.
 
 To run any test suite or individual test with [GodMode turned on](godmode.md), set the value of `test.godmode.enabled` to `true` in `test.properties`.
 
+**NOTE** 
+> GodMode is a tool that aids in browser and email content tests to ensure the source code is as expected and will work across various computers, browsers, user accounts, etc. GodMode is utilized in two main ways: 
```

Comment:
This actually breaks the flow of the document. Imagine someone reading "How to run the test suite with command line" and suddenly having to encounter this.
If anything, this should be placed earlier, e.g. in the "Test configurations" section where it seems to be apt to explain what God mode is and how/why to use it.

In addition, there should not be a repetition of the "God mode is utilized in two main ways" part. Stopping at a brief overview (e.g. the above up to "various computers, browsers, user accounts, etc.") and immediately linking to the God mode docs afterwards would suffice.